sonar-db/src/main/java/org/sonar/db/user/AuthorizationDao.java

```diff
@@ -48,24 +46,25 @@ public AuthorizationDao(MyBatis mybatis) {
     if (componentIds.isEmpty()) {
       return Collections.emptySet();
     }
-    return DatabaseUtils.executeLargeInputs(componentIds, new Function<List<Long>, List<Long>>() {
-      @Override
-      public List<Long> apply(List<Long> partition) {
+    return executeLargeInputs(
+      componentIds,
+      partition -> {
         if (userId == null) {
           return session.getMapper(AuthorizationMapper.class).keepAuthorizedProjectIdsForAnonymous(role, componentIds);
         } else {
           return session.getMapper(AuthorizationMapper.class).keepAuthorizedProjectIdsForUser(userId, role, componentIds);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Make this line start at column 9. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AIndentationCheck)